server.go

```diff
@@ -168,11 +168,15 @@ func (s *QProxyServer) AckMessages(ctx context.Context, in *rpc.AckMessagesReque
 
 func (s *QProxyServer) GetMessages(ctx context.Context, in *rpc.GetMessagesRequest) (resp *rpc.GetMessagesResponse, err error) {
 	start := time.Now()
-	s.m.APIHits.WithLabelValues("GetMessages", in.QueueId.Namespace, in.QueueId.Name).Inc()
+	apiName := "GetMessages"
+	if in.LongPollSeconds <= 0 {
+		apiName = "GetMessagesShortPoll"
+	}
+	s.m.APIHits.WithLabelValues(apiName, in.QueueId.Namespace, in.QueueId.Name).Inc()
 	defer func() {
-		s.m.APILatency.WithLabelValues("GetMessages", in.QueueId.Namespace, in.QueueId.Name).Observe(float64(time.Now().Sub(start)))
+		s.m.APILatency.WithLabelValues(apiName, in.QueueId.Namespace, in.QueueId.Name).Observe(float64(time.Now().Sub(start)))
```

Comment:
Instead of replacing `apiName` it seems that it'd be better to add a label saying it was a short poll? My main concern is that previously the `apiName` matches the actual gRPC api name -- now it doesn't

Reply:
I agree it's not ideal, but adding a label has its problem too: it means adding the label to all other apis e.g. ‘PublishMessage’, which doesn’t has the poll length parameter. It can be confusing too. My feeling is that the apiName just serves as a label to differential different api calls, so an additional name isn't going to cause true problem. It also means we don't need to change current grafana metrics, the new metric will shows up naturally in existing graphs.

Reply:
that being said, I don't have a strong preference here, so if you do believe adding a label is better, I can do it